src/main/java/base/start/model/CommandList.java

```diff
@@ -0,0 +1,22 @@
+package base.start.model;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+public class CommandList extends ArrayList<Command> {
+
+    public void listCommands(HelpMode helpMode) {
+        System.out.println();
+        if(helpMode.equals(HelpMode.EXTENDED)) {
+            System.out.println(this.stream().map(
+                    command -> command.getComandName() + "   |   " +
+                            command.getDescription() + "\n").collect(Collectors.joining())
+            );
+            return;
+        }
+        System.out.println(this.stream().map(
+                command -> command.getComandName() + "    ").collect(Collectors.joining())
+        );
```

Comment:
Is it intended, that you'll have 4 whitespaces remaining at the end of the println call?
If not, could be changed to:

```suggestion
        System.out.println(this.stream().map(Command::getCommandName).collect(Collectors.joining("    "));
```